ci/local/README.md

```diff
@@ -18,12 +18,12 @@ Build and test your local repository using a base gpuCI Docker image
 where:
     -H   Show this help text
     -r   Path to repository (defaults to working directory)
-    -i   Use Docker image (default is gpuci/rapidsai:${NIGHTLY_VERSION}-cuda10.1-devel-ubuntu16.04-py3.7)
+    -i   Use Docker image (default is gpuci/rapidsai:${NIGHTLY_VERSION}-cuda11.5-devel-ubuntu20.04-py3.8)
```

Comment:
What is the difference between developing from `gpuci/rapidsai` and `rapidsai/rapidsai-dev`? It sounds like both are development containers, but I am not familiar with the differences or how to locate that information as a user/developer.

In general, we have a lot of similarly-named containers and I don't know how to identify the purpose of each. The DockerHub pages for each container are very slim on docs, but I don't know of a better source for this kind of information. Should this be raised as a separate issue?

https://hub.docker.com/u/rapidsai
https://hub.docker.com/u/gpuci

Reply:
gpuci images are development images for continuous integration(CI).
gpuci docker images are supposed to same environment as CI. If you want to run CI tests and build locally, these images could be used. These images are available during development cycle of each version of rapidsai.

rapidsai/rapidsai images are not supposed be used for development of rapidsai library itself. It is for development using rapidsai libraries. It contains rapidsai libraries installed. It's available after the each rapidsai release. Not during the development.

rapidsai dockers have 3 flavors: Base, Runtime, Devel. Devel is supposed to be for development using rapidsai libraries. Devel images are not upto date (last updated 2 years ago)
> The RAPIDS images are based on nvidia/cuda, and are intended to be drop-in replacements for the corresponding CUDA images in order to make it easy to add RAPIDS libraries while maintaining support for existing CUDA applications.

Reply:
Not sure of the difference between `gpuci/rapidsai` and `rapidsai/rapidsai-dev`. It seems like they moved all devel containers from `rapidsai` to `rapidsai-dev`.